### PR TITLE
Goosebump story

### PR DESCRIPTION
**Pull request by turnipsundae**
Please review this, @casey-collab

### CODE REVIEW

File: english/goosebumps/goosebumps.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The blue eyes seem so familiar. They remind you of your brother's eyes.
```
Comment (casey-collab) [inline english/goosebumps/goosebumps.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by turnipsundae** [comment]
I've added another line to link back to the main story.

**Comment by turnipsundae** [comment]
I've added another line to the story @casey-collab

File: english/goosebumps/goosebumps.md (hunk 1 above)
Comment (casey-collab) [inline english/goosebumps/goosebumps.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/goosebumps/goosebumps.md (hunk 1 above)
Comment (turnipsundae) [inline english/goosebumps/goosebumps.md]: I've added a link in line 3 @casey-collab

**Pull request closed without merging**